main/project/localization/readme.txt

```diff
@@ -0,0 +1,31 @@
+Checks for installed languages
+==============================
+
+These scripts are only relevant when unsure that the crowdin translations are not up-to-date.
```

Comment:
Suggestion for text:
These scripts are only relevant when unsure whether the crowdin translations are up to date.

Reply:
Ok, and a litte further changed.